pkg/network/http/ebpf_ssl.go

```diff
@@ -198,37 +358,37 @@ func (o *sslProgram) Stop() {
 	o.perfHandler.Stop()
 }
 
-func addHooks(m *manager.Manager, probes map[string]string) func(string) error {
+func addHooks(m *manager.Manager, probes []manager.ProbesSelector) func(string) error {
 	return func(libPath string) error {
 		uid := getUID(libPath)
-		for sec, funcName := range probes {
-			p, found := m.GetProbe(manager.ProbeIdentificationPair{
-				EBPFSection:  sec,
-				EBPFFuncName: funcName,
-				UID:          uid,
-			})
-			if found {
-				if !p.IsRunning() {
-					err := p.Attach()
-					if err != nil {
-						return err
-					}
+
+		for i := range probes {
```

Comment:
```suggestion
		for singleProbe := range probes {
```

Reply:
`i` is the index

Reply:
You are right, can't we operate on the single probe? why do we need the index?

Reply:
changed